graphqlapi/src/main/java/io/stargate/graphql/schema/scalars/IntCoercing.java

```diff
@@ -74,7 +74,7 @@ private DriverTypeT fromBigIntegerSafe(BigInteger i) {
 
   protected void checkRange(Number n, long min, long max) {
     long l = n.longValue();
-    if (l < Byte.MIN_VALUE || l > Byte.MAX_VALUE) {
+    if (l < min || l > max) {
```

Comment:
there was a bug in this range check - both min and max were ignored. However, this function is called by both short and byte checks, so we need to use those arguments. Fixed

Reply:
👍 good catch